### PR TITLE
Update check title and remove icon

### PR DESCRIPTION
**Pull request by ldaneliukas**
It would seem that the ❌ icon is somewhat redundant in most scenarios that we've seen so far. If we look at the whole workflow run, then with or without `fail-on-error` we will still always see a red cross next to the check. This is also true for the checks visible in the Pull Request itself - we already have a ❌ from Github itself. There doesn't seem to be a reason to duplicate this icon with an additional one in the title of the check.

In order to make this a little more useful, I'd propose displaying the total number of tests that succeeded, failed, and were skipped in the title instead. This helps us see a short summary of what happened in the pull request checks section and removes the ❌ icon duplication.

### Workflow summary
Before:
image
image

### Pull  request
Before:
image
After:
image

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -175,15 +174,19 @@ class TestReporter {
 
     const isFailed = results.some(tr => tr.result === 'failed')
     const conclusion = isFailed ? 'failure' : 'success'
-    const icon = isFailed ? Icon.fail : Icon.success
+
+    const passed = results.reduce((sum, tr) => sum + tr.passed, 0)
+    const failed = results.reduce((sum, tr) => sum + tr.failed, 0)
+    const skipped = results.reduce((sum, tr) => sum + tr.skipped, 0)
+    let shortSummary = `${passed} passed, ${failed} failed and ${skipped} skipped `
```
Comment (phjardas) [inline src/main.ts:181]: This could be `const`.

**Comment by tdabasinskas** [comment]
@dorny, is the project still alive?

**Comment by IanMoroney** [comment]
@LDaneliukas , @phjardas , @tdabasinskas , I ran into a similar issue as yourselves where there were features that I needed from this project and looked like it wasn't being updated any time soon, so I took the initiative to fork the project myself and build the enhancements into it with a colleague.
I have also now taken this PR and added the contents into that repo so you can use them :)

https://github.com/phoenix-actions/test-reporting/
https://github.com/marketplace/actions/test-reporting

**Review by karmops — APPROVED** [review]
looks good

**Review by j-catania — APPROVED** [review]

**Review by j-catania — CHANGES_REQUESTED** [review]
could you please revert your dist files please

**Review by j-catania — APPROVED** [review]

**Pull request merged**